aspnet/conceptual-overview/aspnet.rst

```diff
@@ -42,7 +42,27 @@ In summary, with ASP.NET Core you gain the following foundational improvements:
 Application anatomy
 -------------------
```

Comment:
An ASP.NET Core app self-hosts in a console app using `IWebHost`. Its Main method creates and runs the web server using `WebHostBuilder`. 

`WebHostBuilder` is a builder (it follows the builder pattern) that creates a hosting engine for IWebHost and builds the HTTP pipeline. It has methods that define the web server type (`UseKestrel`) and the startup (`UseStartup`) class. In the above example, the Kestrel web server is used, but other web servers can be specified. We'll show more about `UseStartup` in the next section. `WebHostBuilder` provides many optional methods including `UseIISIntegration` for hosting in IIS and IIS Express and `UseContentRoot` for specifying the root content directory. The `Build` and `Run` methods build and run the `IWebHost` that hosts the app and listens for requests.